src/renderer/components/about-app.tsx

```diff
@@ -133,7 +132,8 @@ export default class AboutApp extends React.Component<{}, IState> {
     public copy(): void {
         const data = this.state;
         if (data) {
-            remote.clipboard.write({ text: JSON.stringify(data, null, 4) }, 'clipboard');
+            const data2 = JSON.parse(JSON.stringify(data).split('"clientVersion":').join('"sbeVersion":'));
```

Comment:
What do you think about this?

```suggestion
            const { clientVersion, ...rest } = this.state;
            const data = { ...{ sbeVersion: clientVersion }, ...rest };
            if (data) {
                remote.clipboard.write({ text: JSON.stringify(data, null, 4) }, 'clipboard');
            }
```

Reply:
I recommend this as well.